README.md

```diff
@@ -371,6 +373,11 @@ JSON.stringify(bigObj)
 // ↪TypeError: Do not know how to serialize a BigInt
 ```
 
+### Usage Recommendations
+
+### Coercion
```

Comment:
There's a particular gotcha here, where you do `BigInt(12398764329876236236432243)` to construct a BigInt and  are surprised about the lost precision. This came up in #170. Would it be worth calling out that specifically?